packages/contractkit/src/kit.ts

```diff
@@ -204,6 +209,11 @@ export class ContractKit {
     this.web3.currentProvider.addAccount(privateKey)
   }
 
+  removeAccount(address: Address) {
```

Comment:
Based on my searching, this is only used for testing. If that is true I don't think we should add it to all of our interfaces.